es74x-cloud/src/main/scala/tech/beshu/ror/proxy/es/providers/EsRestClientBasedRorClusterService.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ *     Beshu Limited all rights reserved
+ */
+package tech.beshu.ror.proxy.es.providers
+
+import monix.execution.Scheduler
+import org.elasticsearch.action.admin.indices.alias.get.GetAliasesRequest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4591272)